staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -217,6 +217,8 @@ type ObjectReference struct {
 	// +optional
 	UID types.UID
 	// +optional
+	APIGroup string
```

Comment:
can be add a godoc here and below for ResourceVersion? It's not obvious anymore what to expect in these fields.